client/packages/system/src/Report/components/ReportArgumentsModal.tsx

```diff
@@ -29,11 +29,15 @@ export const ReportArgumentsModal: FC<ReportArgumentsModalProps> = ({
     monthsItemsExpire,
   } = store?.preferences ?? {};
 
+  // default expiry date as now formatted to datepicker
+  const expiryDate = new Date().toISOString().slice(0, 10);
```

Comment:
I wondered if made more sense to have this default coming through with report json schema, but it was much easier to call date here rather than try and add functions to json